users.yaml

```diff
@@ -0,0 +1,87 @@
+swagger: "2.0"
+info: 
+  version: "0.0.1"
+  title: "Preferences"
+  description: "The API lets you manage attributes associated with users (user preferences).\nYou can get any attribute for any user (read only) according to the rights you owe on the user resource. \nThere rights are set up at moment you've subscribed the service exposed by this API."
+  license: 
+    name: "MIT"
+host: "localhost"
+basePath: "/api/v1"
+schemes: 
+  - "https"
+paths: 
+  /Users/{id}: 
+    parameters: 
+      - 
+        name: "id"
+        in: "path"
+        description: "The key of the user to query. The user must be in the organization domain"
+        required: true
+        type: "string"
+        format: "uuid"
+    get: 
+      operationId: "getUser"
+      summary: "Get User"
+      tags: 
+        - "Users"
+      description: "Retrieve one user's preferences (ie. email and/or preferred language)."
+      responses: 
+        200: 
+          description: "The request has succeeded."
+          schema: 
+            $ref: "#/definitions/User"
+definitions: 
+  UserDefinition: 
+    title: "UserDefinition"
+    description: "Describes a user"
+    type: "object"
+    properties: 
+      userName: 
+        type: "string"
+        description: "The user's fiscal number."
+        x-faker: "internet.userName"
+      emails: 
```

Comment:
Maybe. We probably have to think about an onboarding path for existing notifications services. 

Let's suppose [a service has already collected email from users](https://accessosicuro.rete.toscana.it/portal/accessError?targetSeviceUrl=https%3A%2F%2Fservizi.comune.campi-bisenzio.fi.it&errorCode=auth.access.error.message.noURLCertAuth&minRequiredAuthLevel=2&goToUrl=https%3A%2F%2Faccessosicuro.rete.toscana.it%3A443%2Fopensso%2FSSORedirect%2FmetaAlias%2Fidp%3FReqID%3D_b49def1e80d328f724bab2e525918c8d211f510b04%26spEntityId%3Dhttps%253A%252F%252Fservizi.comune.campi-bisenzio.fi.it%252F).

We can store these addresses as an email record associated with a service_id. When the service wants to send a message using the new API it will just provide the service_id (without the user email) and the message could be routed to this specific address (just brainstorming here).